### PR TITLE
Add docs to properties that doesn't have docs.

### PR DESCRIPTION
**Pull request by sairam4123**
I have been adding docs for the properties which doesn't have docs.
This PR is ready for review!

### CODE REVIEW

File: GodotcordActivity.md [hunk 1]
```diff
@@ -3,13 +3,24 @@
 ## GDScript methods/properties
 
 `String state`  
+
+- state of the activity if any
```
Comment (summertimejordi) [inline GodotcordActivity.md:7]: Capitalize your sentences

**Review by summertimejordi — CHANGES_REQUESTED** [review]
Try to follow these guidelines
https://docs.godotengine.org/en/stable/community/contributing/docs_writing_guidelines.html

**Comment by summertimejordi** [comment]
Also you should probably mark this as draft, since it's not finished yet.

File: GodotcordActivity.md (hunk 1 above)
Comment (sairam4123) [reply GodotcordActivity.md:7]: Got it

**Comment by sairam4123** [comment]
Could this be closed as XML Docs are being made?

**Pull request closed without merging**